b2sdk/bucket.py

```diff
@@ -171,6 +171,26 @@ def download_file_by_name(self, file_name, download_dest, progress_listener=None
             url, download_dest, progress_listener, range_
         )
 
+    def get_file_info_by_id(self, file_id: str) -> FileVersionInfo:
```

Comment:
Yeah, I agree it's kindof weird, but more consistent to keep it here as well? As a user I'd just intuitively expect a file operation to be associated with a bucket.

Reply:
We have one case like that already (with download_file_by_id) and we should follow the same pattern - implement the logic in B2Api and then from Bucket call the API method. Do not implement the same thing twice and do not only keep it in the Bucket (because someone might want to get the file info for a fileid without knowing which bucket it is in)

Reply:
Can you give a clearer direction here? I'm not exactly sure what is being asked.

Reply:
please move that method to B2Api and then make a single-line function in Bucket that will `return self.api.get_file_info_by_id(file_id)`

Reply:
Okay, that’s clear. We already have a get_file_info method there. Should we deprecate that one and proceed?

If not, I can just call that existing method.
If so, we could use the [deprecation](https://pypi.org/project/deprecation/) library? (I double checked and the licenses of it and its dependencies are compatible with this library - Apache or MIT)

Reply:
In this project deprecation is handled differently - we have those `from b2sdk.v1 import ...` imports which keep the old interface working.

Do we really need to change the name of the method though? I think it is ok to have `Bucket.get_file_info_by_id(file_id)` call `self.api.get_file_info(fileId)`. Here `Bucket.get_file_info_by_id` will ust be a convenient alias (like `download_file_by_id`)

Reply:
Imo, I would do the oposit, just to be more consequent with the current naming:
* `download_file_by_id` / `download_file_by_name`
* `get_bucket_by_id` / `get_bucket_by_name`

So I would add `get_file_info_by_id` and `get_file_info_by_name` and leave `get_file_info` as an alias.

Reply:
... and deprecate it in v2.

Reply:
Okay I will leave as is then for now - i.e. use get_file_info 👍

The deprecation strategy in this project is a bit opaque (and I personally find the versioning code structure quite confusing), so that was my thought process there.

Reply:
It's a new approach. Thank you for motivating me to write down the rules :)